docs/source/reference/label_format.rst

```diff
@@ -1435,3 +1438,234 @@ To add a :class:`~tensorbay.label.label_sentence.LabeledSentence` label to one d
+
+To add a :class:`~tensorbay.label.label_mask.SemanticMask` label to one data:
+
+    >>> from tensorbay.dataset import Data
+    >>> data = Data("local_path")
+    >>> data.label.semantic_mask = semantic_mask_label
+
+.. note::
+
+   One data can only have one SemanticMask label,
+   See :attr:`Data.label.semantic_mask<tensorbay.dataset.data.Data.label.semantic_mask>` for details.
+
+**************
+ InstanceMask
+**************
+
+InstanceMask is a single channel gray scale image. Different gray-scale values represent different instances.
+For example, if there are more than one person in a photo, in the InstanceMask,
+different people should be divided into different instances.
+In other words, they have the different gray-scale values. It's usually used for instance segmentation task.
+
+Each data can only be assigned with one InstanceMask label.
+
+The structure of one InstanceMask label is like::
+
+    {
+        "remotePath": <str>
+        "info": [
+            {
+                "instanceId": <int>
+                "attributes": {
+                    <key>: <value>
+                    ...
+                    ...
+                }
+            },
+            ...
+            ...
+        ]
+    }
+
+To create a :class:`~tensorbay.label.label_mask.InstanceMask` label:
+
+    >>> from tensorbay.label import InstanceMask
+    >>> instance_mask_label = InstanceMask(local_path = "InstanceMask_image_path")
+    >>> instance_mask_label.all_attributes = {}
+    >>> attributes = {"attribute_name": "attribute_value"}
+    >>> instance_mask_label.all_attributes["instanceId"] = attributes
+    >>> instance_mask_label
+    InstanceMask("InstanceMask_image_path")(
+      (all_attributes): {
+        'instanceId': {
+          'attribute_name': 'attribute_value'
+        }
+      }
+    )
+
+InstanceMask.all_attributes
+===========================
+
+All_attributes is a dictionary that stores attributes for each instance. Each attribute is stored in key-value pairs.
+See :ref:`reference/label_format:attributes` for details.
+
+InstanceMaskSubcatalog
+======================
+
+Before adding the InstanceMask labels to data,
+:class:`~tensorbay.label.label_mask.InstanceMaskSubcatalog` should be defined.
+
+:class:`~tensorbay.label.label_mask.InstanceMaskSubcatalog`
+has categories,
+see :ref:`reference/label_format:attributes information` for details.
+
+To add a :class:`~tensorbay.label.label_mask.InstanceMask` label to one data:
+
+    >>> from tensorbay.dataset import Data
+    >>> data = Data("local_path")
+    >>> data.label.instance_mask = instance_mask_label
+
+.. note::
+
+   One data can only have one InstanceMask label,
+   See :attr:`Data.label.instance_mask<tensorbay.dataset.data.Data.label.instance_mask>` for details.
+
+**************
+ PanopticMask
+**************
+
+PanopticMask is a single channel gray scale image. Each pixel in the image is assigned a semantic label and an instance id.
```

Comment:
In the panoptic mask `png` image, each pixel represents the instance id. The sematic label is in the info of the panoptic mask.